sw/airborne/firmwares/rotorcraft/guidance/guidance_h_ref.h

```diff
@@ -49,29 +50,23 @@ extern float gh_max_speed;
 #define GUIDANCE_H_REF_MAX_ACCEL 5.66
 #endif
 
-/** Update frequency
- */
-#define GH_FREQ_FRAC 9
-#define GH_FREQ (1<<GH_FREQ_FRAC)
-
-#define GH_ACCEL_REF_FRAC 8
-#define GH_SPEED_REF_FRAC (GH_ACCEL_REF_FRAC + GH_FREQ_FRAC)
-#define GH_POS_REF_FRAC (GH_SPEED_REF_FRAC + GH_FREQ_FRAC)
+/** fixedpoint representation: Q26.37 will give a range of
+ * 67e3 km and a resolution of 1.5e-11 m. At a rate of 500Hz,
+ * a ref speed of 7.3e-9 m/s could still update the position.
+ * /
```

Comment:
extra space not closing the comment properly
```suggestion
 */
```

Reply:
Oops! 😅